x-pack/plugins/security_solution_serverless/server/cloud_security/cloud_security_metering_task.ts

```diff
@@ -49,17 +49,24 @@ export const getUsageRecords = (
       assetCountAggregation.min_timestamp.value_as_string
     ).toISOString();
 
-    const creationTimestamp = new Date().toISOString();
+    const creationTimestamp = new Date();
+    const minutes = creationTimestamp.getMinutes();
+    if (minutes >= 30) {
+      creationTimestamp.setMinutes(30, 0, 0);
+    } else {
+      creationTimestamp.setMinutes(0, 0, 0);
+    }
+    const roundedCreationTimestamp = creationTimestamp.toISOString();
 
     const subType =
       cloudSecuritySolution === CLOUD_DEFEND
         ? `${CLOUD_DEFEND}_block_action_enabled_${assetCountAggregation.key_as_string}`
         : cloudSecuritySolution;
 
     const usageRecord: UsageRecord = {
-      id: `${CLOUD_SECURITY_TASK_TYPE}_${cloudSecuritySolution}_${projectId}_${creationTimestamp}`,
+      id: `${CLOUD_SECURITY_TASK_TYPE}_${cloudSecuritySolution}_${projectId}_${roundedCreationTimestamp}`,
       usage_timestamp: minTimestamp,
-      creation_timestamp: creationTimestamp,
+      creation_timestamp: creationTimestamp.toISOString(),
```

Comment:
@CohenIdo I noticed that the creation time is not being rounded on `endpoint` implementation
Are you sure that this one should be rounded as well?

Reply:
I agree, we can keep in with the minutes and the seconds.